plugins/SitesManager/vue/src/SiteWithoutData/SiteWithoutData.vue

```diff
@@ -6,88 +6,244 @@
 
 <template>
   <div>
-      <h1 id="start-tracking-data-header">
-        {{translate('SitesManager_SiteWithoutDataStartTrackingDataHeader')}}
-      </h1>
-      <p v-html="$sanitize(siteWithoutDataDescLine1)"></p>
-      <p v-html="$sanitize(siteWithoutDataDescLine2)"></p>
-      <p>&nbsp;</p>
-
-      <WidgetLoader
-        :widget-params="{module: 'SitesManager', action: 'siteWithoutDataTabs'}"
-        :loading-message="`${translate('SitesManager_DetectingYourSite')}...`"
-      />
-
-      <div class="no-data-footer row">
-        <hr v-if="afterIntroEventContent"/>
-
-        <VueEntryContainer :html="afterIntroEventContent"/>
+    <a id="start-tracking-back"
+       v-if="showMethodDetails"
+       @click.prevent="showOverview();"
+    >
+      <span class="icon-chevron-left"></span>
+      {{ translate('Mobile_NavigationBack') }}
+    </a>
+
+    <h1 id="start-tracking-data-header">
+      {{ headline }}
+    </h1>
+
+    <div id="start-tracking-cta">
+      <a rel="noreferrer noopener" target="_blank" :href="inviteUserLink">
+        <span class="icon-user-add"></span>
+        {{ translate('UsersManager_InviteTeamMember') }}
+      </a>
+      <VueEntryContainer :html="additionalCtaContent" v-if="additionalCtaContent"/>
```

Comment:
Should the inviteUserLink be included in the additionalCtaContent? I ask because VueEntryContainer contains a div, which puts it below the inviteUserLink instead of in line with it as the screenshot on the Jira ticket shows.

Reply:
Ah I See. Guess it's actually unneeded that the `VueEntryContainer` renders an additional div element. I'll try to remove it and see if it breaks anything. Would that be enough for you?

Reply:
I've changed the structure and handling of the CTA buttons a bit. It should now be rendered on the same level

Reply:
Thank you. That fixed the layout issue. For some reason, the styling doesn't appear to be applying correctly, but I can try and deal with that from the plugin.
![image](https://github.com/matomo-org/matomo/assets/11394099/958b6e41-1975-40b7-869c-2550b7e61f5b)